docs/extra/PubisherClientSnippets.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Sure - will do that (and probably a lot of other changes to the actual snippets) in follow-up PRs.